selfdrive/modeld/modeld.cc

```diff
@@ -193,17 +193,18 @@ int main(int argc, char **argv) {
             model_eval_frame(&model, q, yuv_ion.buf_cl, buf_info.width, buf_info.height,
                              model_transform, NULL, vec_desire);
         mt2 = millis_since_boot();
+        float model_execution_time = (mt2 - mt1) / 1000.0;
```

Comment:
any reason this is in seconds?

Reply:
Yes, want as much as possible in SI units.